### DISCUSSION TITLE
When I use faust -a <my.cpp> the <<includeclass>> and <<includeInstrinsic>> tags are not removed

### DISCUSSION THREAD

**Discussion by bassmanitram** [grame-cncm/faust#1013] [Q&A] [answered]
If I use a "null" architecture as shown in the help, these two tags DO not appear in the generated C++. But when I use my own architecture cpp file, they remain in the generated source. Any ideas what I might be doing wrong?

**Comment by sletz** [comment]
Why do you mean by _a "null" architecture_?

**Comment by bassmanitram** [comment]
https://faustdoc.grame.fr/manual/architectures/#minimal-structure-of-an-architecture-file

**Comment by sletz** [comment]
Than I guess your architecture file is incorrect: can you share it ?

**Reply by bassmanitram** [reply]
(Asking the owner of the architecture if I can - however, I think I can "obfuscate" the proprietary stuff anyway ...)

**Reply by bassmanitram** [reply]
[headless.cpp.TXT](https://github.com/grame-cncm/faust/files/15200508/headless.cpp.TXT)

Suitably sanitized to CMA (and renamed with a `.TXT` suffix to get it past GitHub's file-type filter - to use it, remove the `.TXT`)

**Comment by sletz** [comment]
This one works :
[arch3.cpp.txt](https://github.com/grame-cncm/faust/files/15203453/arch3.cpp.txt)

I simply created a new one, then added:
```

<<includeIntrinsic>>

<<includeclass>>
```

Then pasted your code. 

(no idea what happened, could not see problematic hidden character or what ever...)

**Reply by bassmanitram** [reply, marked as answer]
LOL - It was DOS line breaks - I compared yours to mine and that was the difference. Changed them all to unix 0x0A and all works. Thanks so much for the help!

**Comment by sletz** [comment]
OK, thanks.
